EIPS/eip-Claimable_Token.md

```diff
@@ -0,0 +1,162 @@
+
+| Parameter | Description |
+| ---------|-------------|
+| amount | amount to active |
+
+#### activeBalanceOf
+
+Returns balance, active balance and epoch of user account. The epoch is an increasing number which is added by one when account get claimed. Epoch is a part of sign data, verified by both user and provider, prevents double spend and double claim.
+
+| Parameter | Description |
+| ---------|-------------|
+| user | user account address |
+
+#### claim
+
+Service provider (account of issuer) sends micropayment message data params to this function, and recover unsigned message. Then this function checks the message with it's signature (probably using  built-in function 'ecrecover'). If the signature is successfully verified, provider get corresponding token, and user active balance reduce the same ammount. Epoch increase by 1
+
+| Parameter | Description |
+| ---------|-------------|
+| from | which user account to be claimed |
+| credit | how many token should be claimed |
+| epoch | current epoch number |
+| signature| micropayment message signature signed by user|
+
+
+### Events
+
+#### Active
+
+Event emits when user calls active function
+
+| Parameter | Description |
+| ---------|-------------|
+| from | indexed msg.sender |
+| amount | amount to be active|
+
+#### TransferIssuer
+
+Event emits when transferIssuer is called
+
+| Parameter | Description |
+| ---------|-------------|
+| oldIssuer | old issuer/beneficiary |
+| newIssuer | new issuer/beneficiary |
+
+#### Claim
+
+Event emits when claim is called
+
+| Parameter | Description |
+| ---------|-------------|
+| from | user account to be claimed |
+| to | msg.sender, the issuer |
+| epoch | which epoch is based on |
+| credit | how many token be claimed |
+
+## Rationale
+This standards provides a way for user and service provider to establish micropayment channel to make free and repaid payment messages off blockchain. Those payment messages are linked one by one, providing a trace of service consumption and keeps an increasing number of credit, which is how much the payer owned to the service provider.
+
+The service provider can stop serving when payer no longer send new payment message or the payment message describes smaller credit than provider think it should be. In this case, the user can either change provider (assuming provider is dishonest) or accept the make up for the loss in next micropayment message.
+
+The Service can claim token on Ethereum at later time using the data of the largest (probably the latest) signed micropayment message.
+
+## Backwards Compatibility
+This EIP is fully backwards compatible as its implementation extends the functionality of [ERC-20].
+
+## Implementation
+The GitHub repository [Bmail_Token](https://github.com/realbmail/Bmail_token) contains the work in progress implementation.
+
+## Copyright
```

Comment:
Security considerations section is required, and needs to be filled out prior to Last Call, but for Draft you at least need the section placeholder.

```suggestion
## Security Considerations
TBD

## Copyright
```